contracts/smart-account/interfaces/modules/ISecurityPolicyManagerPlugin.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {ISecurityPolicyPlugin} from "./ISecurityPolicyPlugin.sol";
+
+address constant SENTINEL_MODULE_ADDRESS = address(0x1);
+
+interface ISecurityPolicyManagerPluginEventsErrors {
+    event SecurityPolicyEnabled(address indexed scw, address indexed policy);
+    event SecurityPolicyDisabled(address indexed scw, address indexed policy);
+    event ModuleValidated(address indexed scw, address indexed module);
+
+    error SecurityPolicyAlreadyEnabled(address policy);
+    error SecurityPolicyAlreadyDisabled(address policy);
+    error InvalidSecurityPolicyAddress(address policy);
+    error InvalidPointerAddress(address pointer);
+    error ModuleInstallationFailed();
+    error EmptyPolicyList();
+}
+
+/// @title Security Policy Manager Plugin
+/// @author @ankurdubey521
+/// @dev Execution Phase Plugin responsible for enforcing security policies during plugin installation on the smart contract wallet
```

Comment:
I love this style of natspec but since all our code base use `/** **/` it would be better for consistency to stick to it

Reply:
What if we hold a team-wide poll on which style is better, and update all natspecs to the same?